README.md

```diff
@@ -8,3 +8,37 @@ to merge Temporal's execution model, BMPN rendering and native event-sourcing wh
 solution.
 
 See the [**Website**](https://business4s.github.io/workflows4s/) for details and join our [**Discord**](https://bit.ly/business4s-discord) for discussions.
+
+## Visualizing Workflows
```

Comment:
We don't use Readme for docs, no need to add this info here. All the docs are inside `website` dir, we can describe the new url functionality there if you wish. But it can also stay undocumented, in the end some parts of the API will always be undocumented.